src/ImageSharp/Metadata/Profiles/Exif/ExifEncodedStringHelpers.cs

```diff
@@ -27,7 +26,8 @@ internal static class ExifEncodedStringHelpers
 
     // 20932 EUC-JP Japanese (JIS 0208-1990 and 0212-1990)
     // https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding?view=net-6.0
-    private static Encoding JIS0208Encoding => CodePagesEncodingProvider.Instance.GetEncoding(20932);
+    private static Encoding JIS0208Encoding => CodePagesEncodingProvider.Instance.GetEncoding(20932) ??
+                                               throw new InvalidOperationException();
```

Comment:
We should revisit how we use this. https://github.com/dotnet/runtime/issues/31229 says to always use `Encoding`